actionpack/test/dispatch/ssl_test.rb

```diff
@@ -3,13 +3,19 @@
 require "abstract_unit"
 
 class SSLTest < ActionDispatch::IntegrationTest
-  HEADERS = { "Content-Type" => "text/html" }
+  HEADERS = { "content-type" => "text/html" }
```

Comment:
Since this is being returned by `app` I think we should use `Rack::CONTENT_TYPE` here. (I don't see the Content Type header being checked anywhere in the middleware so maybe we don't even need it?)

Reply:
Yup, let's 🔥 it.